src/operator/make_loss-inl.h

```diff
@@ -32,19 +32,19 @@ struct MakeLossParam : public dmlc::Parameter<MakeLossParam> {
   float valid_thresh;
   DMLC_DECLARE_PARAMETER(MakeLossParam) {
     DMLC_DECLARE_FIELD(grad_scale).set_default(1.0f)
-    .describe("gradient scale as a supplement to unary and binary operators");
+    .describe("Gradient scale as a supplement to unary and binary operators");
     DMLC_DECLARE_FIELD(valid_thresh).set_default(0.0f)
-    .describe("regard element valid when x > valid_thresh, this is "
-    "used only in valid normalization mode.");
+    .describe("Clip x to 0 when x <= valid_thresh, where x is the elements in the "
```

Comment:
Might be more intuitive if you could just say "clip each element in the array to 0 when it is less than `valid_thresh`.